components/Payment/installation.rst

```diff
@@ -1,18 +1,11 @@
 Installation
 ============
 
-We assume you're familiar with `Composer <http://packagist.org>`_, a dependency manager for PHP.
-Use following command to add the component to your `composer.json` and download package.
+You can install the component in 2 different ways:
```

Comment:
Every component "Install" chapter start with this "We assume  you're..." sentence. Is there any particular reason to break this convention? You could only add part with downloading component from git to the existing description.